src/contentChunk.js

```diff
@@ -61,3 +61,29 @@ exports.extractStringPayloadIntoChunks = (payload) => {
     stream.end(payload)
   })
 }
+
+export const retrievePayload = (fileId, callback) => {
+  const db = connectionDefault.client.db()
+  if (!db) {
+      const err = new Error(`Payload retrieval failed. Database handle: ${db} is invalid`)
+      return callback(err)
+  }
+
+  if (!fileId) {
+    const err = new Error(`Payload retrieval failed: Payload id: ${fileId} is invalid`)
+    return callback(err)
+  }
+
+  const bucket = new mongodb.GridFSBucket(db)
+
+  let body = ''
+  bucket.openDownloadStream(fileId)
+    .on('error', err => {
+      const error = new Error(`Payload retrieval failed: Error in reading stream: ${err.message}`)
+      return callback(error)
```

Comment:
We can just respond with the error that was thrown, instead of creating a new Error